pkg/network/tracer/connection/kprobe/dump.go

```diff
@@ -0,0 +1,187 @@
+// +build linux_bpf
+
+package kprobe
+
+/*
+#include "../../../ebpf/c/tracer.h"
+*/
+import "C"
```

Comment:
We are trying to avoid importing `C`. If we need to add new `cgo` generated types, then we should do that.

Reply:
done